src/chi-vue/src/constants/types.ts

```diff
@@ -50,6 +50,7 @@ export interface DataTableRow {
     isNestedContentPrintDisabled?: boolean;
   };
   state?: DataTableRowStates;
+  alignmentToParent?: boolean;
```

Comment:
@SergioQCL PR is looking good, only thinking about the naming. What about: expandedParentRow, autoExpandedRow...? 
@josecarlosgonzalezv thoughts?

Reply:
@jllr yeah the naming is the most difficult part of this since it's not strictly expanded. In the end it's just a regular row and the changes are about the alignment of certain elements. I'm thinking on autoExpandedAlignment or something like that.

Reply:
Since it affects the selectable row and aligns all content (checkbox as well), I like the name: `autoExpandedAlignment`